bring-it-up/bring-it-up
# Add ESLint rules to backend and frontend

## Description
#### Summary:
- Add more ESLint rules to the backend to ensure code consistency
- Configure ESLint for frontend (including React linting)

#### Changes:
- Add rules to `.eslintrc` in server
- Add `.eslintrc` to client
- Fix linting errors in server resulting from rule changes
- Fix linting errors in client resulting from new rules

#### How to test this PR:
- Run `npm run lint` in both the `server` and `client` folders. There should be no errors (but there might be warnings)
  
## Checklist
